packages/plugin/webpack/src/WebpackPlugin.ts

```diff
@@ -24,7 +24,7 @@ const BASE_PORT = 3000;
 
 export default class WebpackPlugin extends PluginBase<WebpackPluginConfig> {
   name = 'webpack';
-  private isProd = false;
+  isProd = false;
```

Comment:
I'm not entirely sure how this works in JavaScript, but I wonder if it works to keep this `private` and stub `isProd` in the test.